numpy/distutils/msvccompiler.py

```diff
@@ -15,3 +17,6 @@ def initialize(self, plat_name=None):
             os.environ['lib'] = environ_lib + os.environ['lib']
         if environ_include is not None:
             os.environ['include'] = environ_include + os.environ['include']
+        if platform_bits == 32:
```

Comment:
msvc will use the sse2 code anyway, this just makes it consistent over all of numpy